### DISCUSSION TITLE
请问我命中缓存后 在alova发送请求之前定义的loading我该在什么时机关闭它

### DISCUSSION THREAD

**Discussion by MarshallEriksen-Neura** [alovajs/alova#230] [Q&A]
### 请详细描述bug

请问我命中缓存后 在alova发送请求之前定义的loading我该在什么时机关闭它

### 复现步骤

不是bug

### alova 版本

2.13.1

**Comment by JOU-amjs** [comment]
问题请在Discussion中提问。
loading状态不需要你自己处理的，loading状态会自动改变值

**Reply by JOU-amjs** [reply]
我是不是曲解你的问题了，你可以试看看这边 https://github.com/alovajs/alova/issues/231
